### ISSUE TITLE
[TextFields] Placeholder text color doesn't seem to pass contract requirements on white background

### ISSUE THREAD

**Issue by jverkoey** [jverkoey/mlissues#36]
Testing with: https://webaim.org/resources/contrastchecker/ shows a failing contrast ratio for placeholders.
